enterprise/audit/table.go

```diff
@@ -307,6 +309,59 @@ var auditableResourcesTypes = map[any]map[string]Action{
 		"field":   ActionTrack,
 		"mapping": ActionTrack,
 	},
+	&database.WorkspaceAgent{}: {
+		"id":                         ActionTrack,
+		"created_at":                 ActionIgnore,
+		"updated_at":                 ActionIgnore,
+		"name":                       ActionTrack,
+		"first_connected_at":         ActionIgnore,
+		"last_connected_at":          ActionIgnore,
+		"disconnected_at":            ActionIgnore,
+		"resource_id":                ActionTrack,
+		"auth_token":                 ActionIgnore,
+		"auth_instance_id":           ActionIgnore,
+		"architecture":               ActionIgnore,
+		"environment_variables":      ActionIgnore,
+		"operating_system":           ActionIgnore,
+		"instance_metadata":          ActionIgnore,
+		"resource_metadata":          ActionIgnore,
+		"directory":                  ActionIgnore,
+		"version":                    ActionIgnore,
+		"last_connected_replica_id":  ActionIgnore,
+		"connection_timeout_seconds": ActionIgnore,
+		"troubleshooting_url":        ActionIgnore,
+		"motd_file":                  ActionIgnore,
+		"lifecycle_state":            ActionIgnore,
+		"expanded_directory":         ActionIgnore,
+		"logs_length":                ActionIgnore,
+		"logs_overflowed":            ActionIgnore,
+		"started_at":                 ActionIgnore,
+		"ready_at":                   ActionIgnore,
+		"subsystems":                 ActionIgnore,
+		"display_apps":               ActionIgnore,
+		"api_version":                ActionIgnore,
+		"display_order":              ActionIgnore,
+	},
+	&database.WorkspaceApp{}: {
+		"id":                    ActionIgnore,
+		"created_at":            ActionIgnore,
+		"agent_id":              ActionTrack,
+		"display_name":          ActionTrack,
+		"icon":                  ActionIgnore,
+		"command":               ActionIgnore,
+		"url":                   ActionIgnore,
+		"healthcheck_url":       ActionIgnore,
+		"healthcheck_interval":  ActionIgnore,
+		"healthcheck_threshold": ActionIgnore,
+		"health":                ActionIgnore,
+		"subdomain":             ActionIgnore,
+		"sharing_level":         ActionIgnore,
+		"slug":                  ActionTrack,
+		"external":              ActionIgnore,
+		"display_order":         ActionIgnore,
+		"hidden":                ActionIgnore,
+		"open_in":               ActionIgnore,
+	},
```

Comment:
Review: I'm a bit unsure of what we should track or ignore here. For instance track/ignore app id is a coin-toss for me when comparing to other resources.

Reply:
I'll take a look, but iirc generally you only `track` things that can change. uuids never change and can be ignored.

Reply:
Ah thanks, that clarifies it 👍

Reply:
Gotcha, I made this all ignore for now, I'm thinking we can always unignore later if it makes sense.